### DISCUSSION TITLE
Mass Mailbox Change

### DISCUSSION THREAD

**Discussion by Maeob** [simple-login/app#1058] [Q&A]
Does anyone know how to do a mass change of mailboxes?
I like to have certain types of mail go to different boxes.

**Comment by TheFou** [comment]
Hi,

Totally agree, it is much needed.
Changing each alias manually in case of a mailbox migration would be a real hassle.

**Reply by Maeob** [reply]
The easiest answer to this is allowing us to update via csv (email, enabled, name, active, etc). However, as it stands right now, it will only recognize new addresses in the import process.
